includes/Admin/Dashboard.php

```diff
@@ -433,6 +433,7 @@ public function get_dashboard_settings(): array {
 				'settings'       => '/web-stories/v1/settings/',
 				'pages'          => '/wp/v2/pages/',
 				'publisherLogos' => '/web-stories/v1/publisher-logos/',
+				'categories'     => '/web-stories/v1/web_story_category/',
```

Comment:
Since `web_story_category` is part of a URL path, it should be hyphenated `web-story-category` to follow patterns currently in use (and standard to WordPress).

Reply:
This could probably be shortened to just `categories` since `web-stories` is already part of the path.

Reply:
Ah ok - that’s an existing endpoint.

Reply:
Ideally we do not hardcode `web_story_category` here, but just pass `'taxonomies'     => '/web-stories/v1/taxonomies/',`

Then the dashboard can ping this endpoint to get a list of all hierarchical taxonomies and then display a dropdown for those.

So that this is not hardcoded to just this single taxonomy.